File: 0001-two-sum/0001-two-sum.cs
public class Solution {
 
    public int[] TwoSum(int[] nums, int target) {
        int [] Result=new int[2];
        for(int i=0;i<nums.Length;i++)
        {
            for(int j=i+1;j<nums.Length;j++){
                if(nums[i]+nums[j]==target){
                    Result[0]=i;
                    Result[1]=j;
                }
            }

            }
            return Result;
        }
}

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int secondNum=target-nums[i];
            if(map.containsKey(secondNum)){
                return new int []{map.get(secondNum),i};
            }
            map.put(nums[i],i);
        }
        return null;
       /* int [] res=new int [2];
        for(int i=0;i<=nums.length-2;i++){
            int j=i+1;
            while(j<nums.length){
                if(nums[i]+nums[j]==target){
                    res[0]=i;
                    res[1]=j;
                    break;
                }
                else{
                    j++;
                }

            }
        }
        return res;*/
    }
}